src/routes.js

```diff
@@ -297,6 +475,122 @@ routes.filesSearch = function (arg) {
 };
 
 /**
+ * Create a new file with the contents provided in the request. Do not use this
+ * to upload a file larger than 150 MB. Instead, create an upload session with
+ * upload_session/start.
+ * @function Dropbox#filesUpload
+ * @arg {Object} arg - The request parameters.
+ * @arg {Object} arg.contents
```

Comment:
we should probably add something here. even just a generic "the file contents to be uploaded" is probably better than nothing.